### PR TITLE
amp-ad Cedato AMP Ad provider

### PR DESCRIPTION
**Pull request by cedatoContrib**
The pull request contains implementation for the Cedato Ads for AMP Ads.

The corporate CLA is already signed.

Please review and comment with any questions or concerns.

Best regards

### CODE REVIEW

File: ads/cedato.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    'extraParams',
+  ];
+  validateData(data, requiredParams, optionalParams);
+
+  if (!data || !data.id) {
+    global.context.noContentAvailable();
+    return;
+  }
+
+  const cb = (Math.random() * 10000 | 0);
```
[the remark this answers is not in the document]
Comment (cedatoContrib) [reply ads/cedato.js]: Equivalent to Math.floor, but will be changed

**Comment by cedatoContrib** [comment]
All required changes are done. All automation tests have passed except for Percy test. We do not have access to the fail reason (forbidden message is shown), but we know for sure that no visual changes have been made. Please, grand me access to Percy error details so that I can fix them.

Best regards

**Comment by zhouyx** [comment]
@cedatoContrib Please rebase to latest master to fix the Percy test. Thanks.

**Comment by cedatoContrib** [comment]
We made rebase. All tests have been passed. Thank you.

**Comment by zhouyx** [comment]
@cedatoContrib Looks good! Thanks for the Pull Request. Merged.

**Pull request merged**